src/scripts.js

```diff
@@ -34,13 +34,26 @@ import { changeFormView, displayTrips, displayExpenses, addDestinationSearch, sh
 import Session from './Session'
 
 const session = new Session()
+const loginForm = document.querySelector('#loginForm')
 const userName = document.querySelector('#userName')
 const loginButton = document.querySelector('#login-button')
 const userMenu = document.querySelector('#userMenu')
+const password = document.querySelector('#password')
+const error = document.querySelector('#error')
+loginButton.addEventListener('click', function() {
+  login()
+})
 
-const retrieveUser = () => {
+function login() {
   let loginID = userName.value.split('').splice(8, 2).join('')
```

Comment:
Very cool way of prepping the input values for your logic. Very streamlined conditional!

Reply:
Thank you!